res/css/structures/_FilePanel.pcss

```diff
@@ -42,24 +42,27 @@ limitations under the License.
         display: none;
     }
 
+    /* Overrides for the attachment body tiles */
     .mx_EventTile {
-        /* Overrides for the attachment body tiles */
-        &:not([data-layout="bubble"]) {
-            word-break: break-word;
-            margin-top: 10px;
-            padding-top: 0;
-
-            .mx_EventTile_line {
-                padding-inline-start: 0;
-            }
+        word-break: break-word;
+        margin-top: 10px;
+        padding-top: 0;
+
+        .mx_EventTile_line {
+            padding-inline-start: 0;
+            margin-inline-end: 0;
+        }
+
+        &.mx_EventTile_selected .mx_EventTile_line {
```

Comment:
The class name `mx_EventTile_selected` is applied along with `mx_EventTile`, so combining them with `&` should be fine.